src/Containers.scss

```
@@ -1,4 +1,4 @@
-@import "global-variables";
+@import "global-variables"; // for $pf-v6-global--breakpoint--sm and .pf-v6-u-hidden-on-sm
```

It would probably work with `@use` instead as `@import` is deprecated in Sass
```suggestion
@use "global-variables" as *; // for $pf-v6-global--breakpoint--sm and .pf-v6-u-hidden-on-sm
```

---

So hmm seems my comment is then wrong, this is not for .pf-v6-u-hidden-on-sm

---

Right the `.pf-v6-u-hidden-on-sm` should work without this import, but this import might still give access to it. Try and see :D

---

Right, re-pushed with an updated comment.